### ISSUE TITLE
swipe support

### ISSUE THREAD

**Issue by Gibbz** [maliit/keyboard#17]
Hey guys,

Just wondering if swipe support is planned. I use swipe on the google keyboard on android and find it much faster than having to press each key.
Is there planned support for this feature at all?

**Comment by jpetersen** [comment]
I think that would be a great project.

To work good one needs also good prediction support (to calculate the most proper word one wanted to create swiping). The currently used presage is a bit limited there (even when using the better version which uses marisa instead of sqlite from https://github.com/sailfish-keyboard/presage). It also requires good prediction databases. But yes I think when we can bundle some community around it here it might be possible to create such databases as well.

**Comment by Fuseteam** [comment]
there is prior work for "Swipe" support: https://code.launchpad.net/~michael-sheldon/ubuntu-keyboard/skeyer-prototype/+merge/254008

**Comment by dobey** [comment]
> there is prior work for "Swipe" support: https://code.launchpad.net/~michael-sheldon/ubuntu-keyboard/skeyer-prototype/+merge/254008

I'm not entirely sure what went wrong there, but I am unable to get this converted to a git branch to see what we could use from it. Also the diff itself is over 200MB. Given the amount of changes in both this merge request, as well as those which have happened in maliit (and lomiri-keyboard) over the past 5 years, it would probably better to split things up into smaller changes and perhaps do some of them somewhat differently.

**Comment by Fuseteam** [comment]
yeah the diff is huge, appearently he completely restructured the way plugins work in there, many parts of the diff is removing the 4 familiar qml files
